src/containers/ArtistReleaseContainer.jsx

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
 import { useArtist } from '../hooks/artistHooks';
 import ArtistIDDetail from '../components/artists/ArtistIDDetail';
+import Pagination from '../components/pagination/Pagination';
 import { useParams } from 'react-router-dom';
+import { usePagination } from '../hooks/paginationHook';
 
 
 const ArtistReleaseContainer = () => {
   const { id, name } = useParams();
 
+  const { artistReleases, offset, limit, count, setOffset, loading, currentPage } = useArtist(id);
```

Comment:
i would deal with pages instead of offsets. Then in your service you can convert page to offset before making a request to the API